### PR TITLE
DSFAAP-607_start-page-updates

### PR DESCRIPTION
**Pull request by joseluisgraa**

### CODE REVIEW

File: src/server/home/index.njk [hunk 1]
```diff
@@ -1,42 +1,123 @@
-{% extends 'layouts/page.njk' %}
+{% extends 'layouts/page.njk' %} {% block content %}
```
Comment (hughfdjackson) [inline src/server/home/index.njk:1]: presumably we should put the content block on a new line? 

Did prettifier recommend this?  :thinking:

File: src/server/home/index.njk [hunk 2]
```diff
@@ -1,42 +1,123 @@
...
+          xmlns="http://www.w3.org/2000/svg"
+          width="17.5"
+          height="19"
+          viewBox="0 0 33 40"
+          focusable="false"
+          aria-hidden="true"
+        >
+          <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"></path>
+        </svg>
+      </a>
```
Comment (hughfdjackson) [inline src/server/home/index.njk:118]: @joseluisgraa  - worth using the standard gov uk design system templates with this: 

```jinja2
{% from "govuk/components/button/macro.njk" import govukButton %}

{{ govukButton({
  text: "Start now",
  href: "#",
  isStartButton: true
}) }}
```

Comment (hughfdjackson) [reply src/server/home/index.njk:118]: https://design-system.service.gov.uk/components/button/ <- from here. 

The reason being the usual - we get all of the accessibility benefits out of the box, and don't have to worry about maintaining it as we upgrade the design system.

**Review by hughfdjackson — CHANGES_REQUESTED** [review]

File: src/server/home/index.njk (hunk 1 above)
Comment (joseluisgraa) [reply src/server/home/index.njk:1]: yes it has been done automatically by prettier, not sure why, and not sure how to override these decisions prettier makes

**Review by hughfdjackson — APPROVED** [review]

**Pull request merged**